src/main/scala/scalaz/stream/Process.scala

```diff
@@ -737,8 +753,8 @@ object Process extends ProcessInstances {
   /** The `Process` which emits no values and signals normal termination. */
   val halt: Process0[Nothing] = halt0
 
-  /** Alias for `halt`. */
-  def empty[F[_],O]: Process[F, O] = halt
+  /** Empty emit. Used to signal end of evaluation form `step` to give chance step to be interrupted */
+  val empty: Emit[Nothing] = Emit(Nil)
```

Comment:
Also, the idea of the `empty` method is that it lets you specify the type parameters `F` and `O`, much like the `empty` methods on Scala collections: `List.empty[Int]` or `Map.empty[Int, String]`